food-truck-frontend/src/components/notifications/NotificationD2P.ts

```diff
@@ -0,0 +1,39 @@
+import { Dispatch } from "redux";
+import { NotificationAction, NotificationActionTypes } from "../../redux/notifications/NotificationActions";
+import {getNotifications} from "../../api/Notification";
+import getUserInfo from "../../util/token";
+
+async function updateNotifications(
+  dispatch: Dispatch<NotificationAction>,
+) {
+  // Fetch notifications
+  await getNotifications()
```

Comment:
Unless you need to do more work inline after the asynchronous call, I don't think `wait` is strictly needed, since that basically just tells the function to sit tight until the Promise is resolved.

Reply:
Gotcha. Didn't know what would be considered correct with some of these request functions.